examples/basics/scene/mesh_shading.py

```diff
@@ -21,13 +21,18 @@
 view.camera = 'arcball'
 view.camera.depth_value = 1e3
 
-mesh = Mesh(vertices, faces, color=(.5, .7, .5, 1))
+# Keep the shading off here to manually manage the shading filter below for
+# demonstration purposes. Otherwise, with `shading=flat` or `shading=smooth`,
+# the Mesh class would take care of managing the filter.
+mesh = Mesh(vertices, faces, color=(.5, .7, .5, 1), shading=None)
```

Comment:
Isn't the default `shading=None` in which case the MeshVisual never creates the internal filter?

Reply:
Yes, indeed. My intention was to be explicit but the comment more or less explains it. I'll change this and clarify the comment.

Reply:
I had a try at it in 389d998.